### DISCUSSION TITLE
NTLM is getting retired

### DISCUSSION THREAD

**Discussion by ranjithchev** [tediousjs/tedious#1614] [General]
Hello

As you might already know, MS is planning on retiring NTLM auth and we are currently using tedious driver to connect to sql server from Linux. Once MS retires what is our option? Will tedious add support for kerberos auth? If yes, what is the road map?

**Comment by ranjithchev** [comment]
Any updates here?

**Comment by LightsLegend7276** [comment]
I am also curious about this. We have a an app and mssql db hosted on Linux and my org is requiring kerberos auth and to my knowledge this doesn't work with tedious as of now. Is anyone aware of an alternative package that can use kerberos auth for a nodejs app to mssql db both hosted on linux.
